suite-native/message-system/src/selectors.ts

```diff
@@ -0,0 +1,30 @@
+import { memoize } from 'proxy-memoize';
+
+import { MessageSystemRootState } from '@suite-common/message-system';
+
+export const selectAllActiveMessageIds = memoize((state: MessageSystemRootState) => {
```

Comment:
Compared to desktop implementation, in suite-native there will be all valid messages visible at once, right? In suite-desktop there is always only the one with highest priority visible. But showing all looks more flexible 👍

Reply:
I would say, it will be extremely annoying to show more than 2 banners at once to a user. I believe he will not read it at all.

Reply:
In the end, we decided to change the UI so it does not block the content of the screen. I've changed this PR to a draft. Please wait, until I will rework the UI. I will let you know when the PR is ready again.